frontend/src/scenes/session-recordings/player/snapshot-processing/chrome-extension-stripping.ts

```diff
@@ -18,7 +24,9 @@ interface IsStrippable {
 
 function safelyCheckCSSAttribute(
     node: serializedNodeWithId,
-    attribute: string
+    attribute: string,
+    needles: string[],
+    matchedExtensions: Set<string>
```

Comment:
started passing this set around to keep track of what we're stripping

Reply:
passing a set around because splitting the type narrowing out of these checks made the code pretty ugly